config/routes.rb

```diff
@@ -63,6 +63,8 @@
 
         resources :pages, only: %i[index show create update destroy]
 
+        resources :organizations, only: %i[index]
```

Comment:
Currently the route with nested articles, users, and listings is holding the resources :organizations :show spot. That's the main reason why I return according to the json builder when the show route is visited, to preserve that behavior under v1 when people go looking for things by username.

I think if we were to create similar routes under an organization's id, we could do away with the `params: :username` version of the nested routes, let the suggested "id or slug" lookup happen in the controller action, and have an optional query param to override when the username looks like it could be an id.